docs/coordinates/index.rst

```diff
@@ -346,46 +341,57 @@ To see the list of site names available, use
+.. doctest-remote-data::
 
-    >>> EarthLocation.of_address('1002 Holy Grail Court, St. Louis, MO')
-    <EarthLocation (-26726.98216371, -4997009.8604809, 3950271.16507911) m>
-    >>> EarthLocation.of_address('1002 Holy Grail Court, St. Louis, MO',
-    ...                          get_height=True)
-    <EarthLocation (-26727.6272786, -4997130.47437768, 3950367.15622108) m>
-    >>> EarthLocation.of_address('Danbury, CT')
+    >>> EarthLocation.of_address('1002 Holy Grail Court, St. Louis, MO')  # doctest: +FLOAT_CMP
+    <EarthLocation (-26769.86528679, -4997007.71191864, 3950273.57633915) m>
+    >>> EarthLocation.of_address('Danbury, CT')  # doctest: +FLOAT_CMP
     <EarthLocation ( 1364606.64511651, -4593292.9428273,  4195415.93695139) m>
 
+By default the `OpenStreetMap Nominatim service
+<https://wiki.openstreetmap.org/wiki/Nominatim>`_ is used, but by providing a
+`Google Geocoding API key
+<https://developers.google.com/maps/documentation/geocoding/get-api-key>`_ with
+the ``google_api_key`` argument it is possible to use Google Maps instead. It
+is also possible to query the height of the location in addition to its
+longitude and latitude, but only with the Google queries::
+
+    >>> EarthLocation.of_address("Cape Town", get_height=True)  # doctest: +IGNORE_EXCEPTION_DETAIL
+    Traceback (most recent call last):
+      ...
+    ValueError: Currently, `get_height` only works when using the Google
+    geocoding API...
+
 .. note::
-    `~astropy.coordinates.SkyCoord.from_name`,
-    `~astropy.coordinates.EarthLocation.of_site`, and
-    `~astropy.coordinates.EarthLocation.of_address` are for convenience, and
-    hence are by design relatively low precision. If you need more precise coordinates for an
+    :func:`~astropy.coordinates.SkyCoord.from_name`,
+    :func:`~astropy.coordinates.EarthLocation.of_site`, and
+    :func:`~astropy.coordinates.EarthLocation.of_address` are designed for
+    convenience, not accuracy. If you need accurate coordinates for an
     object you should find the appropriate reference and input the coordinates
     manually, or use more specialized functionality like that in the `astroquery
     <http://www.astropy.org/astroquery/>`_ or `astroplan
     <https://astroplan.readthedocs.io/>`_ affiliated packages.
 
     Also note that these methods retrieve data from the internet to
-    determine the celestial or Earth coordinates. The online data may be
+    determine the celestial or geographic coordinates. The online data may be
     updated, so if you need to guarantee that your scripts are reproducible
     in the long term, see the :doc:`remote_methods` section.
 
 This functionality can be combined to do more complicated tasks like computing
 barycentric corrections to radial velocity observations (also a supported
-high-level |SkyCoord| method - see :ref:`astropy-coordinates-rv-corrs`)::
+high-level |SkyCoord| method - see :ref:`astropy-coordinates-rv-corrs`):
+
+.. doctest-remote-data::
```

Comment:
Ah, you use the `doctest-remote-data`. Maybe this is good enough. This means this doctest block only run in remote data job, right?

Maybe @bsipocz or @saimn remember.

Reply:
This directive indeed applies to the whole code block, see https://github.com/astropy/pytest-doctestplus#remote-data